src/users

```diff
@@ -1 +1,6 @@
+p({"class":"foam.nanos.auth.User","id":1360,"firstName":"intern","lastName":"intern","organization":"intern","department":"intern","email":"intern@intern.net","phone":{"class":"foam.nanos.auth.Phone","number":"1234567890"},"mobile":{"class":"foam.nanos.auth.Phone"},"type":"Business","address":{"class":"foam.nanos.auth.Address","address1":"101 1010101","city":"10010101","postalCode":"k1k1k1","countryId":"CA","regionId":"AB","streetNumber":"101","streetName":"1010101","hours":[]},"password":"TVydKL/E7V0=:a373PUuUTHa5ZONqmO3BbaELOtZuE+pWq2x8pA5t3xtcKjCyXG6s5968zL7/m5R3NolT9V3YpfsgX1fM3BgXyw==","passwordLastModified":"2018-02-23T19:13:55.167Z","businessName":"intern","businessIdentificationNumber":"intern101","group":"business","spid":"nanopay","businessTypeId":0,"businessSectorId":18,"initialEmailedAmount":"$0.00","portalAdminCreated":false,"welcomeEmailSent":true, "emailVerified":true})
+=======
+p({"class":"foam.nanos.auth.User","id":1,"firstName":"admin","middleName":"","lastName":"admin","organization":"","department":"","email":"admin","phone":{"class":"foam.nanos.auth.Phone","verified":false,"number":""},"mobile":{"class":"foam.nanos.auth.Phone","verified":false,"number":""},"type":"","birthday":"1982-07-07T23:12:00.0Z","address":{"class":"foam.nanos.auth.Address","type":"","verified":false,"deleted":false,"structured":true,"address1":"","address2":"","suite":"300","city":"Toronto","postalCode":"M6G2H3","countryId":"CA","regionId":"ON","encrypted":false,"latitude":43.68018341064453,"longitude":-79.4206314086914,"streetNumber":"","streetName":"","hours":[{"class":"foam.nanos.auth.Hours","day":0,"open":true},{"class":"foam.nanos.auth.Hours","day":1,"open":true},{"class":"foam.nanos.auth.Hours","day":2,"open":true},{"class":"foam.nanos.auth.Hours","day":3,"open":true},{"class":"foam.nanos.auth.Hours","day":4,"open":true},{"class":"foam.nanos.auth.Hours","day":5,"open":true},{"class":"foam.nanos.auth.Hours","day":6,"open":true}]},"language":"en","timeZone":"","password":"SEDjby8u5eM=:ION/k78lY9RJx74MyzbWvfAtPGxoL6NRIQ6zXKZJbqHGHJ4LP6XEELayaUqDxfkOeZ8135M4pG6M+KadqqaJ/w==","previousPassword":"","note":"","businessName":"","businessIdentificationNumber":"","issuingAuthority":"","bankIdentificationCode":"","website":"","group":"system","deviceToken":"","businessTypeId":0,"businessSectorId":1,"branchId":"","clearingId":"","status":"","emailVerified":false})
+>>>>>>> 08b9799a2da98b5aab8988b752cbb5531ed22587
```

Comment:
this is a pushed merge conflict. look out for these and make the appropriate changes